source/assets/js/helpers/module.js

```diff
@@ -12,7 +12,7 @@ class EstaticoModule {
 	* @param  {object} options - The options passed as data attribute in the Module
 	*/
 	constructor($element, _defaultData, _defaultOptions, data, options) {
-		this.name = this.constructor.name.toLowerCase();
+		this.name = this.constructor.name;
```

Comment:
The main problem is, that working with `this.name` is only working inside an instance. Which means that all properties must be set at runtime and for every instance again (which can cause a lot of extra operations and will block more memory).

But I don't see a clean solution for your data binding besides adding a `const name = 'module_in_yor_prefered_case';` before the class definition - very similar to the old version of estatico.

(In our team we use `data-js-binding` without the module name. But of course binding to multiple instances is not that clean. On the other hand it solved a lot of problems with nested modules for us.)